Source/WebCore/rendering/RenderTheme.h

```diff
@@ -252,7 +252,7 @@ class RenderTheme {
 #endif
 
 protected:
-    virtual bool canPaint(const PaintInfo&, const Settings&) const { return true; }
+    virtual bool canPaint(const PaintInfo&, const Settings&, const ControlPart) const { return true; }
```

Comment:
No need for `const` if you pass by value.

Reply:
Fixed.